connector-schemas/kafka/table.json

```diff
@@ -22,18 +22,38 @@
                                 "earliest",
                                 "latest"
                             ]
+                        },
+                        "read_mode": {
+                            "type": "string",
+                            "description": "Type of reading behavior for Kafka Source",
+                            "enum": [
+                                "read_committed",
+                                "read_uncommitted"
+                            ]
                         }
                     },
                     "required": [
-                        "offset"
+                        "offset",
+                        "read_mode"
```

Comment:
I believe this is a breaking change that will make existing kafka sources unusable. Can we make it not required and use read_uncommitted as the default value when we use it?

If that's not possible, perhaps it would make sense to include a DB migration that adds this field to existing kafka connections.